index.js

```diff
@@ -118,4 +128,11 @@ function getIt (options, done) {
   });
 }
 
-module.exports = getIt;
+
+
+
+
+var exported = {getIt:getIt, createManifestText:createManifestText, getCWDName:getCWDName};
```

Comment:
I would make this multiple lines, indented, and with a space between the colon and the value for readability.